### PR TITLE
*: support drop stats

### PR DESCRIPTION
**Pull request by lamxTyler**
The syntax is `drop stats tableName`.
PTAL @hanfei1991 @zimulala @winoros

### CODE REVIEW

File: executor/simple.go [hunk 1]
```diff
@@ -320,3 +322,26 @@ func (e *SimpleExec) executeFlush(s *ast.FlushStmt) error {
 	}
 	return nil
 }
+
+func (e *SimpleExec) executeDropStats(s *ast.DropStatsStmt) error {
```
Comment (hanfei1991) [inline executor/simple.go]: We can call `deleteTableStatsFromKV` directly. Just export it.

File: ast/misc.go [hunk 2]
```diff
@@ -41,6 +41,7 @@ var (
 	_ StmtNode = &AnalyzeTableStmt{}
 	_ StmtNode = &FlushStmt{}
 	_ StmtNode = &KillStmt{}
+	_ StmtNode = &DropStatsStmt{}
```
Comment (hanfei1991) [inline ast/misc.go]: We can create a file called ast/stats.go

**Comment by hanfei1991** [comment]
Do we support drop stats DBName.TableName ?

**Comment by lamxTyler** [comment]
@hanfei1991 Yes, it is.

**Comment by hanfei1991** [comment]
LGTM

**Comment by hanfei1991** [comment]
@winoros PTAL

File: parser/parser.y [hunk 3]
```diff
@@ -605,6 +606,7 @@ import (
 	DropTableStmt		"DROP TABLE statement"
 	DropUserStmt		"DROP USER"
 	DropViewStmt		"DROP VIEW statement"
+	DropStatsStmt		"DROP STATS statement"
```
Comment (winoros) [inline parser/parser.y]: Seems it's in lexicographic order? So move it above the `DropTableStme`?

**Pull request merged**